File: simpleTranslate/Category/CALayer+Additions.m
//
//  CALayer+Additions.m
//  simpleTranslate
//
//  Created by Aleksey Anisimov on 26.02.17.
//  Copyright © 2017 vkrotin. All rights reserved.
//

#import "CALayer+Additions.h"

@implementation CALayer (Additions)

-(void)setBorderUIColor:(UIColor*)color
{
    self.borderColor = color.CGColor;
}

-(UIColor*)borderUIColor
{
    return [UIColor colorWithCGColor:self.borderColor];
}


-(UIColor *)shadowUIColor {
    return [UIColor colorWithCGColor:self.shadowColor];
}

-(void)setShadowUIColor:(UIColor *)shadowUIColor {
    self.shadowColor = shadowUIColor.CGColor;
}


@end

File: Swift/simpleTranslate_swift/CALayer+Additions.swift
//
//  CALayer+Additions.swift
//  simpleTranslate_swift
//
//  Created by Aleksey Anisimov on 26.05.17.
//  Copyright © 2017 Aleksey Anisimov. All rights reserved.
//

import Foundation
import QuartzCore
import UIKit

extension CALayer{
    
    var borderUIColor: UIColor {
        set {
            self.borderColor = newValue.cgColor
        }
        
        get {
            return UIColor(cgColor: self.borderColor!)
        }
    }
    
    var shadowUIColor: UIColor {
        set{
            self.shadowColor = newValue.cgColor
        }
        get{
            return UIColor(cgColor: self.shadowColor!)
        }
    }


}
